shell/common/common_benchmarks.sh

```diff
@@ -165,8 +165,42 @@ get_options() {
 
 # Temple functions, re implement in benchmark if needed
 
-benchmark_suite_config() {
-  logger "DEBUG: No specific ${FUNCNAME[0]} defined for $BENCH_SUITE"
+benchmark_suite_start_config() {
+  logger "INFO: preparing confiuration for $BENCH_SUITE"
+
+  #Java is always required regardless of the benchmark to run
+  source_file "$ALOJA_REPO_PATH/shell/common/common_java.sh"
+  init_java
+
+  if [ "$use_hadoop" ]; then
```

Comment:
As I said before, this just tests whether `use_hadoop` has been set to anything other than an empty string. Would rather see `[[ $use_hadoop = 1 ]]`, for example. Same goes for the other variables.